spec/support/saml/response_builder.rb

```diff
@@ -62,6 +62,7 @@ def build_saml_response(authn_context:, account_type:, level_of_assurance:, mult
         multifactor: multifactor
       )
       saml_response = SAML::Responses::Login.new(document_partial(authn_context).to_s)
+      allow(saml_response).to receive(:assertion_encrypted?).and_return(true)
```

Comment:
do you want to return false for SSOe?

Reply:
@annaswims I set this to true in this spec support method because we implicitly assume that we are always dealing with encrypted responses—setting this to true forces the tests that mock out a SAML response to always check `decrypted document`